lib/pattern-preset.js

```diff
@@ -1,10 +1,13 @@
 const presets = {
 
-  'JavaScript': [
-    '.js',
-    '.jsx',
-    '.es6',
-  ],
+  'JavaScript': {
+    pathSubstrings: [
+      '.js',
+      '.jsx',
+      '.es6',
+    ],
+    githubClasses: ['type-javascript'],
+  },
```

Comment:
According to my comment below, keeping the classname here is fine because it's the right place for it.